pkg/daemon/daemon.go

```diff
@@ -209,6 +216,13 @@ func New(
 			return nil, fmt.Errorf("error reading osImageURL from rpm-ostree: %v", err)
 		}
 		glog.Infof("Booted osImageURL: %s (%s)", osImageURL, osVersion)
+
+		ignVersionBytes, err := exec.Command(pathIgnition, "--version").Output()
+		if err != nil {
+			return nil, fmt.Errorf("error getting installed Ignition version: %v", err)
+		}
+		ignVersion = string(ignVersionBytes)
+		glog.Infof("Installed Ignition binary version: %s", ignVersion)
```

Comment:
nit: I think this will read as follows:
```
Installed Ignition binary version: Ignition 0.35.1
```

That isn't really a problem, though if we expect other things to key off this as a version number it may be worth dropping the prefix. Example:

```go
ignVersion = strings.SplitAfter(string(ignVersionBytes), " ")[1]
```

Reply:
:+1: